tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/internal/findandreplace/FindReplaceLogicTest.java

```diff
@@ -38,12 +39,15 @@
 
 import org.eclipse.ui.internal.findandreplace.status.FindAllStatus;
 import org.eclipse.ui.internal.findandreplace.status.FindStatus;
+import org.eclipse.ui.internal.findandreplace.status.FindStatus.StatusCode;
 import org.eclipse.ui.internal.findandreplace.status.InvalidRegExStatus;
 import org.eclipse.ui.internal.findandreplace.status.NoStatus;
 import org.eclipse.ui.internal.findandreplace.status.ReplaceAllStatus;
 
 
 public class FindReplaceLogicTest {
+	private static final int LINE_STRING_LENGTH= 4;
```

Comment:
```suggestion
	private static final int LINE_STRING_LENGTH= LINE_STRING.length();
```

Reply:
Done in https://github.com/eclipse-platform/eclipse.platform.ui/commit/8e8a527bb19dad39d15c797d7cf57a8d889b579b